.changeset/mean-chairs-destroy.md

```diff
@@ -0,0 +1,5 @@
+---
+'@keystone-next/fields': major
+---
+
+The API to configure `virtual` fields has changed to accept a `Field` using the `schema` API exported from `@keystone-next/types` rather than GraphQL SDL. 
```

Comment:
```suggestion
The API to configure `virtual` fields has changed to accept a `field` using the `schema` API exported from `@keystone-next/types` rather than GraphQL SDL. 
```